bin/src/_main.rs

```diff
@@ -326,6 +329,39 @@ pub async fn _main(
         false => None,
     };
 
+    let tailer_source = match (config.log.tailer_cmd, config.log.tailer_args) {
```

Comment:
So I got kind of turned around with how this is set up - and I feel it requires too much domain knowledge. This could just be a misunderstanding on my part. 

Instead of this being wrapped around two configs that are set in a novel way that is new to the agent (by a brand new outside library) but if we wrapped it around if these tailers are defined for a given platform? This is all basically dead code for linux/macos but there is no indication as such. If possible I'm normally against over commenting but a comment around here explaining this I think would be welcome.

Reply:
the source will be None if these params are not configured. meaning - the source is optional. by default it is not enabled on non-windows platforms (as windows installation package comes with bundled agent config file that has these options enabled).
It is standard approach in agent, see line 512.

Reply:
Gunna approve I guess my one ask would be a comment saying where these are set, but i'm not gunna hold up the PR for that.